Makefile

```diff
@@ -625,17 +625,14 @@ $(OBJDIR)/ceed.pc : pkgconfig-prefix = $(prefix)
 	    -e "s:%libs_private%:$(pkgconfig-libs-private):" $< > $@
 
 ifeq ($(filter install,$(MAKECMDGOALS)),install)
-  CPPFLAGS += $(if $(ASAN),$(AFLAGS)) -DCEED_JIT_SOUCE_ROOT_DEFAULT="\"$(abspath $(DESTDIR)$(includedir))/\""
+  $(OBJDIR)/interface/ceed-jit-source-root-default.o : CPPFLAGS += -DCEED_JIT_SOUCE_ROOT_DEFAULT="\"$(abspath $(includedir))/\""
 else
-  CPPFLAGS += -DCEED_JIT_SOUCE_ROOT_DEFAULT="\"$(abspath ./include)/\""
+  $(OBJDIR)/interface/ceed-jit-source-root-default.o : CPPFLAGS += -DCEED_JIT_SOUCE_ROOT_DEFAULT="\"$(abspath ./include)/\""
 endif
 
-$(OBJDIR)/interface/ceed-jit-source-root-default.o: .FORCE
-.FORCE:
+$(OBJDIR)/interface/ceed-jit-source-root-default.o : $(if $(filter install,$(MAKECMDGOALS)),.FORCE)
 
-ifeq ($(filter install,$(MAKECMDGOALS)),install)
-	$(MAKE) $(OBJDIR)/interface/ceed-jit-source-root-default.o
-endif
+.FORCE:
```

Comment:
I'm not sure what you intended here. What I don't like about this is that it changes `lib/libceed.so` and I'm considering whether we should link `build/libceed.so` for installation, so that it doesn't interfere with the in-place libceed.so. I'm thinking of a workflow where someone runs `make test && make install && build/t001-ceed`.

Reply:
I was trying to force the rebuild of the one specific file with the new constant. I'm fine with making a separate version for both `libceed.so`s

Reply:
I believe this works with the latest version (just pushed).

Reply:
I think that's macos using heirloom `make`. Pushed a variable rename that should appease it.

Reply:
It looks like the library isn't being built on macos jobs? I'm not seeing an obvious reason why that would be true

Reply:
Yeah, this line in the output is missing
`make: 'lib' with optional backends: /cpu/self/avx/serial /cpu/self/avx/blocked`

Reply:
It appears that invoking `make` or `make lib` does absolutely nothing in the macos CI environment.

Reply:
they use GNU Make 3.81

Reply:
Which is from **April 2006**!!

Reply:
I think it's erroring early when dealing with the "simple" `:= $(shell ...)` evaluation, but I can't reproduce with make-3.81 on Linux. (I keep it specifically to debug such issues because Apple refuses to update its antiques.)